app/src/main/java/org/simple/clinic/home/patients/PatientScreenRepository.kt

```diff
@@ -0,0 +1,17 @@
+package org.simple.clinic.home.patients
+
+import io.reactivex.Observable
+import org.simple.clinic.home.patients.illustration.HomescreenIllustration
```

Comment:
Is there a reason the `illustration` package is a sub-package of `patients`? We might be using it in the `PatientsScreen`, but from a domain modelling perspective, the two are unrelated.

If we don't have a reason for putting this here, we should move this to a top-level package under `org.simple.clinic`.

Reply:
done